src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorInitializeEndpoint.cs

```diff
@@ -21,6 +27,12 @@ public Task<InitializeResult> HandleRequestAsync(InitializeParams request, Razor
         capabilitiesManager.SetInitializeParams(request);
         var serverCapabilities = capabilitiesManager.GetInitializeResult();
 
+        var telemetryReporter = requestContext.GetRequiredService<ITelemetryReporter>();
+
+        telemetryReporter.ReportEvent("initialize", Severity.Normal, new
```

Comment:
I'm not sure if its a problem worth worrying about, but `intiialize` happens every time the server starts, which is every time a solution is opened, but the values in `LanguageServerFeatureOptions` can only change when VS starts. Perhaps we should use a static bool to ensure we only report this once per VS session, to avoid noisy telemetry?